### PR TITLE
Autoprefixer support

### PR DESCRIPTION
**Pull request by drugoi**
Sorry for big PR, but in future autoprefixer  save many hours of development.
If you have any questions, I will be happy to answer them.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -23,6 +23,17 @@ module.exports = function(grunt) {
         ext: '.css'
       }
     },
+	  autoprefixer: {
+	    options: {
+				browsers: ['last 2 versions', 'Opera >= 15', 'Chrome >= 4', 'Firefox >= 16', 'Safari >= 4']
```
Comment (XhmikosR) [inline Gruntfile.js]: Indentation is off here. (tabs to spaces)

Comment (XhmikosR) [inline Gruntfile.js]: Does specifying IE >= 10 change anything?

Comment (drugoi) [inline Gruntfile.js]: @XhmikosR `IE >= 10` will not change anything, because -ms prefix is dropped in this (and future) releases of IE.

Comment (XhmikosR) [inline Gruntfile.js]: Ah, makes sense.

**Comment by XhmikosR** [comment]
You should squash 2-3 of those patches and keep like 3 separate patches in the branch; it'll look cleaner.

**Pull request closed without merging**